sos/plugins/openstack_novajoin.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2018 Red Hat, Inc., David Vallee Delisle <dvd@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin
+import os
+
+
+class OpenStackNovajoin(Plugin):
+    """OpenStack Novajoin
+    """
+    plugin_name = "openstack_novajoin"
+    profiles = ('openstack', 'openstack_undercloud')
+
+    def setup(self):
+        if self.get_option("all_logs"):
+            self.add_copy_spec([
+                "/var/log/novajoin/"
+            ])
```

Comment:
Both `add_copy_spec()` and `add_cmd_output()` accept either a single string, or a list of strings - please don't wrap single items in `[...]` as it causes things to spill over multiple lines unnecessarily.